src/dswx_sar/defaults/dswx_ni.yaml

```diff
@@ -70,7 +70,7 @@ runconfig:
             mgrs_collection_database_file:
 
         primary_executable:
-            product_type: DSWX_NI
+            product_type: dswx_ni
```

Comment:
I'm not sure why you are trying to change this? To have consistency with the DSWx-S1, I think we should keep as it is.(https://github.com/opera-adt/DSWX-SAR/blob/fca03ed9a0ec1665e6e1340629dcd1abe99370eb/src/dswx_sar/defaults/dswx_s1.yaml#L64)